pkg/daemon/fake/manager.go

```diff
@@ -123,6 +126,89 @@ func (t *testMgr) GetExistingServices() (map[string]struct{}, error) {
 	return svcsMap, nil
 }
 
+func (t *testMgr) EnsureServiceIsRemoved(name string, managedOnly bool) error {
```

Comment:
These methods on the fake manager are required for testing purposes. They are exact copies of those upon the real manager - if anyone has suggestions on how to avoid this, please let me know.